internal/command/migrate_to_v2/volumes.go

```diff
@@ -23,8 +23,11 @@ func (m *v2PlatformMigrator) validateVolumes(ctx context.Context) error {
 	m.usesForkedVolumes = numMounts != 0
 
 	volsPerProcess := map[string]int{}
-	for _, m := range m.appConfig.Mounts {
-		for _, p := range m.Processes {
+	for _, mount := range m.appConfig.Mounts {
+		if len(mount.Processes) == 0 {
+			mount.Processes = m.appConfig.ProcessNames()
+		}
```

Comment:
```suggestion
```

Reply:
still left a call to processNames there but without the side effect because validation was missing the case of empty processes list too

Reply:
double check please